DB/src/main/java/io/deephaven/db/v2/QueryTable.java

```diff
@@ -1860,7 +1860,7 @@ private Table snapshotInternal(Table tableToSnapshot, boolean doInitialSnapshot,
         final SnapshotInternalListener listener = new SnapshotInternalListener(this, lazySnapshot, tableToSnapshot,
                 result, resultLeftColumns, resultRightColumns, resultIndex);
 
-        if (doInitialSnapshot) {
+        if (!isRefreshing() || doInitialSnapshot) {
```

Comment:
I hate the idea of making `snapshot` have different semantics for static tables. I'm not convinced we should merge this.

Reply:
Ahh, yes, I see. Then we just need to fix the documentation. Closing.